yt_dlp/extractor/tiktok.py

```diff
@@ -607,6 +607,7 @@ def _parse_aweme_video_web(self, aweme_detail, webpage_url, video_id, extract_fl
                 'like_count': 'diggCount',
                 'repost_count': 'shareCount',
                 'comment_count': 'commentCount',
+                'bookmark_count': 'collectCount',
```

Comment:
```suggestion
                'save_count': 'collectCount',
```

Reply:
Bonus points if you can find the equivalent mobile API response key so that we can extract this in `_parse_aweme_video_app` as well

EDIT: I found some dumped API responses I had laying around. The mobile API key is `collect_count`